lms/static/scripts/frontend_apps/components/LMSFilePicker.js

```diff
@@ -138,7 +148,16 @@ export default function LMSFilePicker({
           error={dialogState.error}
         />
       )}
-      {dialogState.state === 'authorizing' && (
+      {dialogState.state === 'authorizing' && authorizationAttempted && (
+        <ErrorDisplay
+          message=<Fragment>
```

Comment:
I noticed chrome parsed it a bit funny syntax-wise but since the linter was ok I didn't read to much into it. Thanks for the pointer. Seems to be happy with that change.